draft-ietf-dnsop-domain-verification-techniques.md

```diff
@@ -127,7 +127,7 @@ Random Token: a random value that uniquely identifies the DNS domain verificatio
 
 ## TXT Record
 
-DNS TXT records are the RECOMMENDED method of doing DNS-based domain verification. The provider constructs the validation domain name by prepending a provider-relevant prefix followed by "-challenge" to the domain name being validated (e.g. "\_foo-challenge.example.com").
+The RECOMMENDED method of doing DNS-based domain verification uses DNS TXT records. The provider constructs the validation domain name by prepending a provider-relevant prefix followed by "-challenge" to the domain name being validated (e.g. "\_foo-challenge.example.com").
```

Comment:
There are many ways to use TXT for verification, just see appendix A.  This one is the only one we recommend.